docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,847 @@
+   * :ref:`Clone the SRW App from GitHub <DownloadSRWApp>`
+   * :ref:`Check out the external repositories <CheckoutExternals>`
+   * :ref:`Set up the build environment <SetUpBuild>`
+   * :ref:`Build the executables <BuildExecutables>`
+   * :ref:`Download and stage data <Data>`
+   * :ref:`Optional: Configure a new grid <GridSpecificConfig>`
+   * :ref:`Generate a regional workflow experiment <GenerateForecast>`
+      * :ref:`Configure the experiment parameters <UserSpecificConfig>`
+      * :ref:`Load the python environment for the regional workflow <SetUpPythonEnv>`
+   * :ref:`Run the regional workflow <RocotoRun>` 
+   * :ref:`Optional: Plot the output <PlotOutput>`
+
+.. _AppOverallProc:
+
+.. figure:: _static/FV3LAM_wflow_overall.png
+
+    *Overall layout of the SRW App Workflow*
+
+
+.. _HPCstackInfo:
+
+Install the HPC-Stack
+========================
+
+.. Attention::
+   Skip the HPC-Stack installation if working on a `Level 1 system <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_ (e.g., Cheyenne, Hera, Orion, NOAA Cloud).
+
+**Definition:** :term:`HPC-Stack` is a repository that provides a unified, shell script-based build system and builds the software stack required for `UFS <https://ufscommunity.org/>`_ applications such as the SRW App. 
+
+Background
+----------------
+
+The UFS Weather Model draws on over 50 code libraries to run its applications. These libraries range from libraries developed in-house at NOAA (e.g. NCEPLIBS, FMS, etc.) to libraries developed by NOAA's partners (e.g. PIO, ESMF, etc.) to truly third party libraries (e.g. NETCDF). Individual installation of these libraries is not practical, so the `HPC-Stack <https://github.com/NOAA-EMC/hpc-stack>`__ was developed as a central installation system to ensure that the infrastructure environment across multiple platforms is as similar as possible. Installation of the HPC-Stack is required to run the SRW App.
+
+Instructions
+-------------------------
+Users working on systems that fall under `Support Levels 2-4 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_ will need to install the HPC-Stack the first time they try to run applications (such as the SRW App) or models that depend on it. Users can either build the HPC-stack on their local system or use the centrally maintained stacks on each HPC platform if they are working on a Level 1 system. For a detailed description of installation options, see :ref:`Installing the HPC-Stack <InstallBuildHPCstack>`.  
+
+After completing installation, continue to the next section.
+
+.. _DownloadSRWApp:
+
+Download the UFS SRW Application Code
+======================================
+The SRW Application source code is publicly available on GitHub. To download the SRW App, clone the release branch of the repository:
+
+.. code-block:: console
+
+   git clone -b develop https://github.com/ufs-community/ufs-srweather-app.git
```

Comment:
Should I wait until the release branch has been cut to update it though (in another PR)? Otherwise, anyone following these instructions will run into trouble.

Reply:
Totally fine to leave it as develop for now until the release branch is cut. However, then maybe change the sentence above to say "develop branch" rather than "release branch"?